### DISCUSSION TITLE
custom page title not working with Redis caching

### DISCUSSION THREAD

**Discussion by andrewlynch** [solidusio/solidus#4112] [Troubleshooting]
I’ve set a custom page title on product pages using `self.title` however, when enabling caching with Redis it defaults original page title. Is there a fix for this?

**Comment by jarednorman** [comment]
Is this with the stock frontend?

**Reply by andrewlynch** [reply]
Yes, I'm using the stock frontend

**Reply by jarednorman** [reply]
I don't follow what you mean by using `self.title`. Can you provide some more information or context about what you're doing/how you're doing it?

The title tag in the head is (I think) not cached, so are you referring to the actual `h1` on the product page? That's cached based on locale, current pricing options and the product, so as long as the product's `updated_at` changes, your change should apply. I don't follow what you're doing that this wouldn't be working.
